Dockerfile

```diff
@@ -50,6 +50,18 @@ ENV QUILT_PATCHES=debian/patches
 
 COPY build_backport.sh /scripts/
 
+# Retrieve binary packages for gcc-4.9 from toolchain test PPA
+RUN sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+RUN sudo apt-get update \
+  && sudo apt-get \
+    -y \
+    --download-only \
+    -o Dir::Cache::Archives="/build/" \
+    install gcc-4.9
+RUN sudo rm /etc/apt/sources.list.d/ubuntu-toolchain-r-test-trusty.list
```

Comment:
What if the install fails?

Reply:
this isn't actually an install, just a download... if the download fails, we fail to get these packages and the whole process stops :)

this isn't good enough though, I've had to remove these binary packages from the repository and right now I've got the trusty branch downloading the gcc-4.9 packages directly from this PPA, and our `pip install` stuff switches to gcc-4.9 and then switches back.  which is pretty hacky.

what i'm probably going to have to do is fetch the source packages from the ppa and apply the 9 patches that debian came up with for adding retpoline support to gcc 4.9, and then adding gcc-4.9-with-retpoline to our repository...